resources/js/components/Modal.vue

```diff
@@ -49,9 +49,9 @@
 <template>
     <transition name="modal">
         <div class="z-50 fixed pin overflow-y-scroll modal-mask" @click="handleClicks">
-            <div class="bg-contrast rounded shadow-lg max-w-md mx-auto my-10 p-5 modal-container">
+            <div class="bg-contrast relative rounded shadow-lg max-w-md mx-auto my-10 p-5 modal-container">
```

Comment:
Modal needs to be relative to use absolute position for loader background shade.